### PR TITLE
Incorrect SMS sent to Mom on Clinic line

### PR DESCRIPTION
**Pull request by gsvr**
JIRA Ref: https://praekelt.atlassian.net/browse/NDOH-76

SMS copy currently: "Congratulations on your pregnancy. You will now get free SMSs about MomConnect. You can register for the full set of FREE helpful messages at a clinic."

Correct copy is: "Welcome to MomConnect. For more info or to log a complaint or compliment, please dial _134_550#. If you ever want to stop getting SMSs dial _134_550*1#"

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/clinic.js [hunk 1]
```diff
@@ -105,8 +105,8 @@ go.app = function() {
             return self.im.outbound.send({
                 to: self.contact,
                 endpoint: 'sms',
-                content: "Congratulations on your pregnancy. You will now get free SMSs about MomConnect. " +
-                         "You can register for the full set of FREE helpful messages at a clinic."
+                content: "Welcome to MomConnect. For more info or to log a complaint or compliment, " +
+                        "please dial *134*550#. If you ever want to stop getting SMSs dial *134*550*1#"
```
Comment (imsickofmaps) [inline src/clinic.js]: This should be `$` wrapped and context added for dial in lines I think

**Comment by gsvr** [comment]
@imsickofmaps Recheck?

**Comment by imsickofmaps** [comment]
:+1: and noted that we need to move to AAT range for all QA configs: https://github.com/praekelt/go-ndoh/issues/93

**Pull request merged**